homeassistant/components/aurora_abb_powerone/__init__.py

```diff
@@ -1 +1,58 @@
 """The Aurora ABB Powerone PV inverter sensor integration."""
+
+# Reference info:
+# https://s1.solacity.com/docs/PVI-3.0-3.6-4.2-OUTD-US%20Manual.pdf
+# http://www.drhack.it/images/PDF/AuroraCommunicationProtocol_4_2.pdf
+#
+# Developer note:
+# vscode devcontainer: use the following to access USB device:
+# "runArgs": ["-e", "GIT_EDITOR=code --wait", "--device=/dev/ttyUSB0"],
+
+import asyncio
+import logging
+
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.core import HomeAssistant
+
+from .const import DOMAIN
+
+PLATFORMS = ["sensor"]
+
+_LOGGER = logging.getLogger(__name__)
+
+
+async def async_setup(hass: HomeAssistant, config: dict):
+    """Set up the Aurora ABB PowerOne component."""
+    return True
+
+
+async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry):
+    """Set up Aurora ABB PowerOne from a config entry."""
+    hass.data.setdefault(DOMAIN, {})
+    _LOGGER.debug("async_setup_entry got user config data=%s", entry.data)
```

Comment:
Let's take the debug line out before merge

Reply:
Agree. Done.